lib/scanner/index.js

```diff
@@ -32,28 +32,20 @@ exports.scan = function(config) {
 
     ///
     function filterTechs_(techs) {
-        return _.filter(techs, function(tech) {
-            return config.isTargetPath(tech.path) && !config.isExcludedPath(tech.path);
-        });
+        return techs.filter(tech => config.isTargetPath(tech.path) && !config.isExcludedPath(tech.path));
     }
 };
 
 /**
  * @param {Object[]} techs
- * @returns {Tech[]}
+ * @returns {Promise<Tech[]>}
```

Comment:
Docs think differently https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#type-application